packages/client/src/connection.ts

```diff
@@ -1,11 +1,15 @@
-import { AsyncLocalStorage } from 'async_hooks';
 import * as grpc from '@grpc/grpc-js';
-import { temporal } from '@temporalio/proto';
-import { TLSConfig, normalizeTlsConfig } from '@temporalio/internal-non-workflow-common';
+import { normalizeTlsConfig, TLSConfig } from '@temporalio/internal-non-workflow-common';
+import root, { temporal } from '@temporalio/proto';
+import { AsyncLocalStorage } from 'async_hooks';
+import type { Root } from 'protobufjs';
 import { defaultGrpcRetryOptions, makeGrpcRetryInterceptor } from './grpc-retry';
 
 export type WorkflowService = temporal.api.workflowservice.v1.WorkflowService;
-export const { WorkflowService } = temporal.api.workflowservice.v1;
+
+// use `root.lookupService` because imported `WorkflowService` doesn't have `create` method
+const WorkflowService = (root as unknown as Root).lookupService('temporal.api.workflowservice.v1.WorkflowService');
```

Comment:
Can you explain why doesn't it have `create`? It doesn't seem to be an issue with the other protos.